packages/sdc/components/03-organisms/side-navigation/side-navigation.css

```
@@ -157,15 +163,29 @@
   outline-style: solid;
   outline-color: var(--ct-outline-light);
 }
-.ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail > .ct-menu__item__link, .ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail > .ct-menu__item__link:active, .ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item.ct-menu__item--active-trail > .ct-menu__item__link:visited {
+.ct-side-navigation.ct-theme-light .ct-side-navigation__menu .ct-menu__item .ct-menu__item__link:active {
   background-color: var(--ct-side-navigation-light-link-child-active-background-color);
   border-color: var(--ct-side-navigation-light-link-child-active-border-color);
   color: var(--ct-side-navigation-light-link-child-active-color);
 }
```

Stylelint (17.14.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)